### PR TITLE
chore: delete presences for unavailable websites

### PR DESCRIPTION
**Pull request by ImRodry**
This PR deletes a series of presences either fully or by removing certain URLs that were invalid from the allowed URLs list. This is to ensure the best quality of the presences on the store so as to not have discontinued websites there.
Here are all the removed presences so far:

- 20XX.info - website domain is now for sale
- AnimePill - website domain is now for sale
- Aniwatch - website has shut down and no longer provides a secure connection
- Anontpp - website URL is invalid
- Asphalt Events - DNS is no longer connected
- Astr.io - website now redirects to a different service (author confirmed it is no longer valid)
- Bargrooves - website sends to a default Apache2 Ubuntu page.
- BetterDiscord - this website does not follow our [guidelines](https://docs.premid.app/en/dev/presence/guidelines) regarding illegal websites.
- Boo.rip - presence quality was extremely low and not up to our standards, and the website also appears to be either unregistered or empty
- BotsBase - DNS is not registered
- BugMeNot - this website does not follow our [guidelines](https://docs.premid.app/en/dev/presence/guidelines) regarding illegal websites.
- CartoonCrazy - website has been offline and does not provide a secure connection anymore
- ChickenFM - domain is now for sale
- Cisco Networking Academy - removed all URLs except for the main one as they did not work anymore/were redirects to cisco's main website
- Comic Space - website is now a default test website
- Custom Status - removed new.eggsy.xyz and updated descriptions to use the main URL
- DevianArt - removed www.eclipsefeedback.com as the author confirmed it no longer works
- Divine Discord Bot List - website leads to a scam website/domain for sale
- Dizipub - DNS not registered/scam wesbite/domain for sale/I have no clue what's going on here
- fastani.net - inaccessible website
- Flackless - website said goodbye
- Game-linter - domain is for sale
- Gamepedia - website now redirects to fandom.com
- Pastr - domain is now for sale
- Punch! Fansub - website is not accessible (403 error)
- StatusBot - domain is now for sale
- SteamPrices.com - website has shut down
- TKA E-Sports - website cannot be reached
- Wanflix - DNS is not registered
- Wikimedia Commons - removed test website as it hasn't been operational for a while now
- WoxeBot - subdomain no longer exists - service was removed
- Aniflix - domain change from `*.tv` to `.cc`
- Anime Music Quiz - `www.*` redirects to `*`
- Anime-D - added nsfw tag as there is nsfw ads on the website
- AnimeSaturn - domain is no longer working
- AnimeX - domain is for sale
- Animelizm - added nsfw tag as there is nsfw ads on the website
- Bundle Haber - service renamed to Budle and new domain
- Binarysearch - `*.io` redirects to `*.com`
- Waze - domain redirects to `www.*`
- WaniKani - domain redirects to `www.*`
- Destiny - domain redirects to `www.*`
- DevTR - invalid DNS
- dizilab - `*.pw` redirects to `*.io`
- Neox Scanlator - `*.com` redirects to `*.net`
- Nettruyen - `nettruyen.com` redirects to `nettruyenpro.com`
- niconico - `live2.*` redirects to `live.*`
- osu!ripple - `vinse.*` shows a default nginx page
- PartydoosMediaGroup - poor quality code 
- pays.host - new domain & poor code
- RLSwaps - redirects to `rl.supply`
- ReaperX - Invalid DNS
- Repeat - Invalid DNS
- Rizo-Host - Invalid DNS
- Foster - invalid DNS
- Memesena - Invalid DNS
- Milk and Cookies - Invalid DNS
- Nakama Mangas - Invalid DNS
- Bundle - Invalid DNS
- Kael - Invalid DNS
- Kahve BOT - Invalid DNS
- KeyFM - Invalid DNS
- Kimi no Na wa Wikia - Invalid DNS
- Liebe Schnee Hiver - Invalid DNS
- Love Live! Wikia - Invalid DNS
- Livida Blog - Invalid DNS
- Hangout Hotel - Invalid DNS
- HepBoat - Invalid DNS
- LabyDesigns - Invalid DNS
- Union Mangas - `unionleitor.top` invalid dns
- South Park - `*.lat` redicts to `*.com`
- Vidstreaming - `gogo-stream.com` redirects to `gogoplay1.com`
- HBO GO - updated domains
- Invidious - Outdated domains
- Jisho - `classic.*` is invalid
- Komiku - Removed invalid subdomains
- Koya - `koyamie.com` redirects to `koya.gg`
- Loritta - Removed useless subdomains
- MangaDex - `*.org` redirects to `*.cc`
- MangaKakalot - Removed deprecated domain
- Manganato - presence removed in favor of MangaKakalot as they work for the same websites

If you see a presence on this list that you think shouldn't be removed, please leave a review!
Co-authored by @EncryptedDev

### CODE REVIEW

**Review by Slowlife01 — DISMISSED** [review]

File: websites/S/South Park/dist/metadata.json [hunk 1]
```diff
@@ -6,16 +6,15 @@
   },
   "url": [
     "www.southparkstudios.com",
-    "www.southpark.lat",
-    "southpark.cc.com"
```
Comment (Slowlife01) [inline websites/S/South Park/dist/metadata.json:10]: I believe these are used in some countries

Comment (ImRodry) [reply websites/S/South Park/dist/metadata.json:10]: I don't even know why these were removed *cough* @EncryptedDev

Comment (EncryptedDev) [reply websites/S/South Park/dist/metadata.json:10]: Oops, add `*.co.uk` as well then

Comment (ImRodry) [reply websites/S/South Park/dist/metadata.json:10]: Actually I do: these all redirect to the first URL, just like the newly added one, so will remove that as well

**Comment by ImRodry** [comment]
Rebased and addressed conversation

File: websites/I/Invidious/dist/metadata.json [hunk 2]
```diff
@@ -12,16 +12,8 @@
...
-    "invidious.gcc-project.de",
-    "invidious.glie.town",
-    "invidious.site",
     "invidious.snopyta.org",
-    "invidious.toot.koeln",
-    "invidious.tube",
-    "invidious.xyz",
     "vid.mint.lgbt",
     "yewtu.be"
   ],
```
Comment (Slowlife01) [inline websites/I/Invidious/dist/metadata.json:19]: Needs version bump

File: websites/D/DeviantArt/dist/metadata.json [hunk 3]
```diff
@@ -12,7 +12,6 @@
     "portfolio.deviantart.com",
     "shop.deviantart.com",
     "www.deviantartsupport.com",
-    "www.eclipsefeedback.com",
```
Comment (Slowlife01) [inline websites/D/DeviantArt/dist/metadata.json:15]: Needs version bump

**Comment by ImRodry** [comment]
All bumped

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**